src/core/connection.h

```diff
@@ -208,6 +208,16 @@ typedef union QUIC_CONNECTION_STATE {
         //
         BOOLEAN DisableVneTp : 1;
 #endif
+
+        //
+        // Whether to use QTIP on sends for this connection.
+        //
+        BOOLEAN UseQTIP : 1;
+
+        //
+        // Whether or not the app explicitly set the UseQTIP flag.
+        //
+        BOOLEAN AppDidSetQTIP : 1;
```

Comment:
Once you convert the parameter to a setting, I don't think this will be needed anymore; possibly not `UseQTIP` either, considering you store the state in the `Route` too.

Reply:
This should be removed still.

Reply:
Done.